lib/inspec/cli.rb

```diff
@@ -39,6 +39,9 @@ class Inspec::InspecCLI < Inspec::BaseCLI
   class_option :disable_user_plugins, type: :string, banner: '',
     desc: 'Disable loading all plugins that the user installed.'
 
+  class_option :telemetry, type: :boolean,
+    desc: 'Allow or disable telemetry', default: true
```

Comment:
I'm pretty sure this should default to false - we're doing opt-in on other Chef products, IIRC...

Reply:
Very weak suggestion, Consider naming this `--enable-telemetry` - I always like verb phrases